### DISCUSSION TITLE
ui5-avatar-group doesn't show button when wrapped by flexBox

### DISCUSSION THREAD

**Discussion by JosephineSu** [UI5/webcomponents#2630] [Q&A] [answered]
Hi all,
 
I  wanna use avatarGroup to display align down avatar items. If AvatatGroup is wrapped by FlexBox, the button is not visible when the available space is less than the width required to display all avatars. 
Could you please help figure out why this happens and how to solve ?

![image](https://user-images.githubusercontent.com/38159212/103732629-c0483580-5022-11eb-83b6-5f269721f245.png)
![image](https://user-images.githubusercontent.com/38159212/103732643-c63e1680-5022-11eb-947b-c842f98f1fb7.png)

**Comment by ilhan007** [comment, marked as answer]
Hello @JosephineSu

does your FlexBox container/parent shrink according to the available width?

This is the first thing you have to check, because if the parent in not shrinking, the AvatarGroup won't shrink and overflow as well.

**Reply by JosephineSu** [reply]
Hi @ilhan007 
FlexBox did shrink when I resize the window, but AvatarGroup didn't.

**Reply by ilhan007** [reply]
Then I think it is better to open an issue and create a small example so we can quickly reproduce it.

You can use codesandbox as a quick sample platform and here we have a [small example](https://codesandbox.io/s/71r1x5o51q) that will help you start up quickly
 or a live URL to your app that we can access.

Note: just update the dependency to 1.0.0-rc.11 (it uses RC.9 currently) to test against the latest


<img width="346" alt="Screenshot 2021-01-08 at 10 58 18 AM" src="https://user-images.githubusercontent.com/15702139/103995174-c401c680-51a0-11eb-89b1-374147747428.png">

**Reply by JosephineSu** [reply]
Thank you @ilhan007. I change styles, In FlexBox container use a Div to wrap AvatarGroup, and set the style of Div:  {overflow: auto; width: 100%} , Then it woks. No need to open issue.
